httpserver/swarm.cpp

```diff
@@ -150,13 +150,14 @@ static all_swarms_t apply_ips(const all_swarms_t& swarms_to_keep,
 
 void Swarm::apply_swarm_changes(const all_swarms_t& new_swarms) {
 
-    all_cur_swarms_ = apply_ips(new_swarms, all_cur_swarms_);
+    all_valid_swarms_ = apply_ips(new_swarms, all_valid_swarms_);
 }
 
 void Swarm::update_state(const all_swarms_t& swarms,
+                         const std::vector<sn_record_t>& decommissioned,
```

Comment:
Not sure if I am only looking at a partial diff, but I can't see where this new `decommissioned` variable is being used in `update_state`?

Reply:
That's a bug, good catch! Should be adding them to `all_funded_nodes_`, otherwise they wouldn't be tested and won't have a chance to come back online! Fixed now.